app/reducers/article.js

```diff
@@ -12,26 +12,19 @@ function getInitialState() {
   return {
     article: {},
     currentArticle: null,
-    nextArticle: null,
-    topics: topicJsonMock.results
+    nextArticle: null
```

Comment:
> The client obviously will not have any idea what article ID will be returned.

This is represented currently in the URL. `(http://localhost:3001/app/#/article/0)` where 0 is the articleId. If we are planning on changing this I would like to know the reason why. 

> and the server will return either a full URL, or an URL hash # fragment, or an article ID

The client should expect one response, an article payload. `GET` article and `POST` highlights should be two discrete actions. We can represent them as one user action, but they should not be bundled together into a single request and response.

>the next article received from the endpoint "highlightQueue"

The next article should come from articleQueue. This is possibly just semantics here. 😄 

... All in all, sounds like we really need some documentation about the way that articles will be fetched and highlights will be processed. I'm definitely open to suggestions but let's not let user flow drive API design.